index.html

```diff
@@ -275,9 +275,48 @@ <h2 id="principal-title" class="color-blue fonts">About Myself
 
     <div class="divisor">
     </div>
+
+    <section id="form-section">
+      <div id="contact-container">
+        <div>
+          <h2 class="white">Contact me</h2>
+          <p class="white">If you have an application you are interested in developing, a feature that you need built or
+            a project that needs coding.
+            I’d love to help with it! </p>
+        </div>
+        <form class="form2" id="form" action="https://formspree.io/f/mqkoqdwg" method="POST">
```

Comment:
- Kindly make sure that :+1: 
    - Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
        * `action` value should be similar to `https://formspree.io/f/{form_id}`
        * `method` value equals `POST`
   -  Please check the `network` tab in your browser dev tools to confirm that the form is submitted successfully with valid input values.
- All input fields are wrapped in a single `form` element.
Please make sure you implement correctly all the points I have  listed above and check   in your browser if your is  successfully submitted when you click on the button submit after completing the form :+1: